server/__temp__index.js

```diff
@@ -0,0 +1,134 @@
+const express = require('express');
```

Comment:
Good routes, consider exporting them to a controller file using express.Router()